### ISSUE TITLE
No gold pickup 

### ISSUE THREAD

**Issue by joekienzle** [kolton/d2bot-with-kolbot#204]
Neither my cBaal leader or leecher are picking up any good. I turned of FieldId and added the correct line to nip file. Any ideas why they don't pick up money? Maybe because they have stashed enough?

**Comment by mboufos** [comment]
your pickit has gold pick ?

**Comment by joekienzle** [comment]
Yes it did. Sorry I found the issue. They both simply had full gold both in the stash and on them.

**Comment by mboufos** [comment]
:D put gamble on heheh

**Comment by joekienzle** [comment]
Is what I'm actually planning to do to gamble for amus for crafting. :D 

How do I control when to stop gambling?

**Comment by mboufos** [comment]
Config.Gamble = true;
	Config.GambleGoldStart = 2000000;  <--start
	Config.GambleGoldStop = 1500000; <--stop

**Comment by LeMule** [comment]
Its literally two lines after setting the gambling flag.

**Comment by joekienzle** [comment]
Sorry I meant: how do I combine crafting and gambling so that the gambling script stops to gamble when a suitable base item is found? Or is it gambling for many base items for crafting until the money runs out?

And what kind of amulet is the gamble script keeping? Or does that depend on my pickit? And if an amulet is in my pickit is it used for crafting or simply kept?

**Comment by mboufos** [comment]
gambling is gambling  cubing is cubing there are no conflicts on those 2 :)

**Comment by joekienzle** [comment]
OK so that means the script will gamble until the money runs out? 

When I gamble for amulets what kind are kept?

**Comment by LeMule** [comment]
Gamble keeps stuff thats in the pickit.
Amulets found through pickit entries are not used for crafting. A runner
keeps any necessary items for crafting, aside from pickit entries.

Alot of Kolbot is well documented in plain English, however. I'd recommend
at least reading through the relevant libs you're trying to use (the
comments at the top, not the code, unless you're comfortable with that),
since you'd have answered all your questions relatively quickly on your own.

On Sun, Mar 5, 2017, 5:39 AM Johannes Kienzle <notifications@github.com>
wrote:

Sorry I meant: how do I combine crafting and gambling so that the gambling
script stops to gamble when a suitable base item is found? Or is it
gambling for many base items for crafting until the money runs out?

And what kind of amulet is the gamble script keeping? Or does that depend
on my pickit? And if an amulet is in my pickit is it used for crafting or
simply kept?

—
You are receiving this because you commented.


Reply to this email directly, view it on GitHub
<https://github.com/kolton/d2bot-with-kolbot/issues/204#issuecomment-284216617>,
or mute the thread

**Comment by mboufos** [comment]
Gamble keep money 
cubing keep what ever you put to cube :P i cant understand what confused you :D hehehe

**Comment by joekienzle** [comment]
I'm coming from etal. There they have built in to gamble for a base amulet for crafting, but anyhow I'll read through the code. I guess no one understands my question. It's actually pretty simple though:

If you want to gamble an amulet as a base for crafting, how do you set that up?

**Comment by joekienzle** [comment]
Read through the code and still couldn't find an answer to my question. I guess it's just simply not possible.

**Comment by mboufos** [comment]
there is a list  like 2 lines from gamble :P 
	// List of item names or classids for gambling. Check libs/NTItemAlias.dbl file for other item classids.
	Config.GambleItems.push("Amulet");
	Config.GambleItems.push("Ring");
	Config.GambleItems.push("Circlet");
	Config.GambleItems.push("Coronet");

here and you put what to gamble :)

**Comment by LeMule** [comment]
Nobody understood it because you asked a multitude of questions that were
not relevant to your actual "simple" question.

It's sad that you just said "I'll read through the code", implying you have
some kind of technical knowledge yet are asking the most simple,
spoonfed-answered questions.

This is, after all, an issue list, and not a help-desk. You may suggest new
functionality or contribute, if you'd like.

**Comment by joekienzle** [comment]
Your right, the conversation emerged from the issue. I'll stop asking questions here. Thanks anyway.

**Comment by mboufos** [comment]
come to irc :P will be faster to get your answer  :D

**Comment by joekienzle** [comment]
What is the server and channel, please? Or am I too stupid to find it again? :D

**Comment by mboufos** [comment]
check this
https://github.com/kolton/d2bot-with-kolbot/wiki/IRC-bot:-standard

and put channel #d2bs

noone is stupid :D relax hehehehe we are more familiar to it so we think some code is obvious :P
